objectivec/docs/docs.readme.md

```diff
@@ -0,0 +1,78 @@
+# ONNX Runtime Objective-C API
+
+ONNX Runtime provides an Objective-C API.
+It can be used from Swift with a bridging header.
+
+The public headers are located [here](https://github.com/microsoft/onnxruntime/tree/master/objectivec/include).
+One can import them individually, or import the entire API with [onnxruntime.h](https://github.com/microsoft/onnxruntime/blob/master/objectivec/include/onnxruntime.h).
+
+## Example Usage in Objective-C++
+
+```objectivec
+#import <Foundation/Foundation.h>
+
+#import <onnxruntime.h>
+
+// Adds two numbers using ONNX Runtime.
+float add(float a, float b) {
+  // We will run a simple model which adds two floats.
+  // The inputs are named `A` and `B` and the output is named `C` (A + B = C).
+  // All inputs and outputs are float tensors with shape [1].
+  NSString* const kAddModelPath = @"/path/to/add.ort";
+
```

Comment:
Would it be helpful to mention the path to the model used in the unit tests so that could be used if someone wanted to actually execute the example?

Reply:
moving the example code into the examples repo where it will hopefully be more easily runnable.
https://github.com/microsoft/onnxruntime-inference-examples/pull/7